### PR TITLE
Merge kato-core into clouddriver-core.

### PR DESCRIPTION
**Pull request by duftler**
Merge kato-web into clouddriver-web.
kato-core goes away completely.
kato-web still exists to collect all the kato-* subprojects while we merge them one-by-one.
Tested by invoking GCE and AWS operations and querying for task status.
Also worked through merging kato-kubernetes into clouddriver-kubernetes to confirm it is straightforward (those changes are not part of this PR).

### CODE REVIEW

**Comment by gregturn** [comment]
I was hoping I could get https://github.com/spinnaker/clouddriver/pull/192 merged before such a sweeping change.

**Comment by duftler** [comment]
No worries. This won't be merged yet. Let's get your PR in and #213 before this one.
Once those two are merged, I'll update this one.

**Comment by duftler** [comment]
Strange. The two tests that failed on travis pass for me locally.

**Comment by gregturn** [comment]
Travis has been fickle for me too.

**Comment by duftler** [comment]
I was able to reproduce this test failure on a fresh ubuntu vm. Still trying to figure out what the difference in setup is between my mac and the vm/travis that's aggravating the test.

**Comment by duftler** [comment]
Rebased to incorporate Azure LB's and fix to unit test. Travis is happy now.
Will need to rebase again once #192 and #219 are merged.

**Comment by duftler** [comment]
Rebased and ready for review.
@ajordens please review.

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy [hunk 1]
```diff
@@ -52,10 +55,10 @@ import java.security.Security
   CloudFoundryConfig,
   AzureConfiguration,
   SecurityConfig,
-  com.netflix.spinnaker.kato.Main,
   com.netflix.spinnaker.mort.Main,
   com.netflix.spinnaker.oort.Main
 ])
+@ComponentScan(["com.netflix.spinnaker.kato.config"])
```
Comment (ajordens) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy]: Gotcha.

I reckon we can remove this once all the kato-XXX providers are migrated as the @Import(XYZConfig) should add an appropriate component scan.

**Comment by ajordens** [comment]
LGTM

Is this waiting on anything else? I'd be happy with it going down.

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy (hunk 1 above)
Comment (duftler) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy]: Yup, that was my thinking.

**Comment by duftler** [comment]
Nope, good to go.

**Pull request merged**